app/views/appointment_mailer/confirmation.html.erb

```diff
@@ -1,36 +1,137 @@
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Dear <%= @appointment.first_name %>,</p>
+<h1 style="color: #0B0C0C !important;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-weight: 700;font-size: 20px;line-height: 1.111111111;padding: 15px 0;">
+  You've successfully booked an appointment
+</h1>
 
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Thank you for booking an
-appointment with Pension Wise. We can confirm your appointment will be at <%=
-@appointment.start_at.to_s(:govuk_date) %>. The appointment will be with <%= @appointment.guider.name %>.</p>
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
+  Dear <%= @appointment.first_name %>,
+</p>
 
-<h3 style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif">Preparing for your appointment</h3>
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
+  Thank you for booking a telephone guidance appointment with Pension Wise.
+  Your appointment is confirmed for:
+</p>
 
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">To make your appointment as useful
-as possible, it would be helpful to know the following information:</p>
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
+  Date
+  <br>
+  <strong class="emphasize" style="font-size: 24px;">
+    <%= @appointment.start_at.to_s(:govuk_date) %>
+  </strong>
+</p>
 
-<ul style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px">
-  <li>the value of your pension pot(s) - check your pension paperwork or ask your provider</li>
-  <li>an estimate of your State Pension - call the Future Pension Centre on 0345 600 4274 for a State Pension statement</li>
-  <li>any special arrangements attached to your pension pot, eg a guaranteed annuity rate or a guaranteed pot value at a certain time - ask your provider</li>
-</ul>
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
+  Start time
+  <br>
+  <strong class="emphasize" style="font-size: 24px;">
+    <%= @appointment.start_at.to_time.to_s(:govuk_time) %>
+  </strong>
+</p>
+
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
+  Appointment lasts
+  <br>
+  <strong class="emphasize" style="font-size: 24px;">
+    <%= ((@appointment.end_at - @appointment.start_at) / 1.minutes).round %> minutes
```

Comment:
Ah I've just noticed this isn't quite right.

Although we have 70 minute appointments in the database, the customer's appointment is only 45 minutes. We allow for more time so the guider has time to prepare either side.

It might be worth just hardcoding this in for now...